newIDE/app/src/EventsSheet/SelectionHandler.js

```diff
@@ -5,13 +5,17 @@ import values from 'lodash/values';
 export type InstructionsListContext = {|
   isCondition: boolean,
   instrsList: gdInstructionsList,
+  quick?: any,
+  domEvent?: any,
```

Comment:
I'm a bit confused, do we need both? Can we just store one? Do we even need these?

Reply:
likely a flowtype error. Maybe quick should really be called quickAddMenuTarget?

Reply:
Instead of quick I would really prefer `inlineInstructionEditor`. So something like `inlineInstructionEditorAnchorEl` (to mimic the naming of `inlineEditingAnchorEl`)

Reply:
fixed now